apis/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Generic API Server",
-   "version": "v1.9.0"
+   "version": "v0.0.0"
```

Comment:
i thought we are continuing to follow the versioning of k8s releases. Are we starting federation releases from v0.0.0?